logging/v5_6/logging-5-6-configuration.adoc

```diff
@@ -9,3 +9,7 @@ toc::[]
 include::snippets/logging-crs-by-operator-snip.adoc[]
 
 include::snippets/logging-supported-config-snip.adoc[]
+
+include::modules/logging-loki-retention.adoc[]
+
+include::modules/logging-loki-alerts.adoc[]
```

Comment:
Isn't the Loki Alerts section wrong here? It is a 5.7-only feature.

Reply:
Yep, threw it in there so it had somewhere to render a preview before I created the 5.7 sub-folder.